backend/app/api/v1/endpoints/telco/telcoJobs.py

```diff
@@ -50,23 +50,83 @@ async def jobs(
             ),
             status_code=422,
         )
-
+    if offset and not size:
+        raise HTTPException(400, f"offset {offset} specified without size")
+    elif not offset and not size:
+        size = 10000
+        offset = 0
+    elif not offset:
+        offset = 0
     results = await getData(start_date, end_date, size, offset, "telco.splunk")
+
     jobs = []
-    if len(results["data"]) >= 1:
+    if "data" in results and len(results["data"]) >= 1:
         jobs = results["data"].to_dict("records")
-
     response = {
         "startDate": start_date.__str__(),
         "endDate": end_date.__str__(),
         "results": jobs,
-        "total": 0,
-        "offset": 0,
+        "total": results["total"],
+        "offset": (offset + size) if size else 0,
     }
 
     if pretty:
         json_str = json.dumps(response, indent=4)
         return Response(content=json_str, media_type="application/json")
 
     jsonstring = json.dumps(response)
-    return jsonstring
+    return response
+
+
+@router.get(
+    "/api/v1/telco/filters",
+    summary="Returns data to build the filter",
+    description="Returns a list of jobs in the specified dates of requested size. \
+            If not dates are provided the API will default the values. \
+            `startDate`: will be set to the day of the request minus 5 days.\
+            `endDate`: will be set to the day of the request.",
```

Comment:
If you decide to use my other comment on the default value wording, it should also be applied here.

Reply:
Done!